tty.c

```diff
@@ -2449,6 +2516,38 @@ tty_colours_bg(struct tty *tty, const struct grid_cell *gc)
 	tc->bg = gc->bg;
 }
 
+static void
+tty_colours_us(struct tty *tty, const struct grid_cell *gc)
+{
+	struct grid_cell	*tc = &tty->cell;
+	char			s[32];
+
+	/* Check if this is a 24-bit or 256-colour colour. */
+	if (gc->us & COLOUR_FLAG_RGB || gc->us & COLOUR_FLAG_256) {
+		if (tty_try_colour(tty, gc->us, "58") == 0)
+			goto save_us;
+		/* Should not get here, already converted in tty_check_us */
+		return;
+	}
+
+	/* Is this an aixterm bright colour? */
```

Comment:
Since we can not specify underline color in aixterm, should this block simply be removed?